<ui version="4.0" >
 <class>StreetsDialog</class>
 <widget class="QDialog" name="StreetsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Streets</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QListView" name="listView" >
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView" />
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Find:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findlineEdit" />
       </item>
       <item>
        <widget class="QToolButton" name="cleartoolButton" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addpushButton" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Street:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addlineEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findlineEdit</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>addlineEdit</tabstop>
  <tabstop>addpushButton</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>cancelpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>StreetsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>502</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel" >
     <x>315</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleartoolButton</sender>
   <signal>clicked()</signal>
   <receiver>findlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>502</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
